### PR TITLE
Tom_Shandrikov

### PR DESCRIPTION
**Pull request by ashandrikov**

### CODE REVIEW

File: gradlew.bat [hunk 1]
```diff
@@ -64,14 +61,28 @@ echo location of your Java installation.
 
 goto fail
 
+:init
```
Comment (ArtemNikolaev1) [inline gradlew.bat:64]: Why did you change these files?

**Comment by ashandrikov** [comment]
Not on purpose. Is it bad? How to fix?

чт, 8 июл. 2021 г., 13:02 ArtemNikolaev1 ***@***.***>:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Java Core June 2021
+
+## *Tom Shandrikov*
```
Comment (NikolaevArtem) [inline README.md:3]: Please merge with master

File: src/main/java/hometask_1/Main.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+public class Main {
+    public static void main(String[] args) throws Exception {
+
+        try {
+            for (String str : args) {
+
+                if (str.equalsIgnoreCase("error")) {
+                    System.out.println("\u001B[31m" + "Alarm!" + "\u001B[0m");
+                    throw new RuntimeException();
```
Comment (NikolaevArtem) [inline src/main/java/hometask_1/Main.java:11]: Exception shouldn't be thrown, is't not an exceptional case, it's expected behaviour

File: src/main/java/hometask_1/Main.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package hometask_1;
+
+public class Main {
+    public static void main(String[] args) throws Exception {
+
+        try {
+            for (String str : args) {
+
+                if (str.equalsIgnoreCase("error")) {
+                    System.out.println("\u001B[31m" + "Alarm!" + "\u001B[0m");
```
Comment (NikolaevArtem) [inline src/main/java/hometask_1/Main.java:10]: Better use constants for escape strings

**Comment by NikolaevArtem** [comment]
Duplicate
